cogs/stars.py

```diff
@@ -3,15 +3,16 @@
 from nextcord.ext.tasks import loop
 
 
+STARS_CHANNEL_ID = 884441198520045570
+
```

Comment:
If it's used in several classes or other places in the file, it sometimes makes sense to make it global.

in general, I think it's best to keep things in the smallest scope that it may need to be used in, though.

If anything, I'd change the other files to be more like this, but I don't think it's necessary to modify it, as it already works and I don't think it's too inconsistent.

Personally, I don't think the IDs and stuff should be in the code at all, rather in a configuration or environment file. In this case I felt it would be better for it to be self-contained though, in order to not require refactoring multiple files or changing env stuff I don't have access to.

Reply:
> Personally, I don't think the IDs and stuff should be in the code at all, rather in a configuration or environment file
My first thought as well, but for that, I'd need to change alot of other stuff as well, which I thought was outside the scope of this PR.

I will revert this for now and do the above mentioned, if approved, in a following pull request.